### PR TITLE
EPMFARMATS-12631

### PR DESCRIPTION
**Pull request by overdd**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "js-training",
+  "version": "1.0.0",
+  "description": "test",
+  "main": "index.js",
+  "scripts": {
+    "democode": "node  index.js",
```
Comment (SiarheiRylach) [inline package.json:7]: two whitespaces :)

File: post.js [hunk 2]
```diff
@@ -0,0 +1 @@
+console.log("This is post-democode message")
```
Comment (SiarheiRylach) [inline post.js:1]: ';' should be in the end of the line.

File: pre.js [hunk 3]
```diff
@@ -0,0 +1 @@
+console.log("This is pre-democode message")
```
Comment (SiarheiRylach) [inline pre.js:1]: ';' should be in the end of the line.

**Comment by overdd** [comment]
Issues are fixed

**Pull request merged**